contracts/eden/src/actions/induct.cpp

```diff
@@ -32,4 +32,41 @@ namespace eden
 
       inductions.update_profile(induction, new_member_profile);
    }
+
+   void eden::inductvideo(eosio::name account,
+                          uint64_t id,
+                          std::string video)
+   {
+      require_auth(account);
+      inductions inductions{get_self()};
+      auto induction = inductions.get_induction(id);
+      eosio::check(inductions.is_endorser(id, account),
```

Comment:
I think we should check if the member is active here. in case that the member is banned after an induction process starts...

Reply:
Done